.gitignore

```diff
@@ -5,5 +5,6 @@
 /.idea
 /coverage
 /.vscode
+/test-output/**/*
```

Comment:
isn't `/test-output/**/*` the same as `/test-output` Same for the .npmignore file.